server/userAuth.js

```diff
@@ -72,6 +68,23 @@ router.use((req, res, next) => {
   res.redirect('/login')
 })
 
+function isAuthorized(user) {
+  const [{value: userEmail = ''} = {}] = user.emails || []
+  const [userDomain] = userEmail.split('@').slice(-1)
+  const checkRegexEmail = (() => {
+    const domainsArray = [...domains]
+    for (const d in domainsArray) {
```

Comment:
What about a `for ... of` here instead of `for ... in` ? Since the intention is to always return a boolean, using a `reduce` would be great too.